src/runtime/cli/config/configuration-qemu.toml.in

```diff
@@ -56,6 +56,13 @@ firmware = "@FIRMWAREPATH@"
 # For example, `machine_accelerators = "nosmm,nosmbus,nosata,nopit,static-prt,nofw"`
 machine_accelerators="@MACHINEACCELERATORS@"
 
+# Qemu seccomp sandbox feature
+# comma-separated list of seccomp sandbox features to control the syscall access.
+# For example, `seccompsandbox= "on,obsolete=deny,elevateprivileges=deny,spawn=deny,resourcecontrol=deny"`
+# Please note, enable this feature may lead some performance cost, you may enable
+# /proc/sys/net/core/bpf_jit_enable to reduce the impact.
+#seccompsandbox="@DEFSECCOMPSANDBOXPARAM@"
```

Comment:
Could we have a boolean flag instead, with a sane default? With Kata we have a fairly well defined device model, we should be able to build the right set of seccomp rules for our use cases.
I'm suggesting this because I feel our config file is already very complex...

Reply:
The seccomp rule is for Qemu process itself, not for Kata. 
Yes with Kata we have a fairly well defined device model but we cannot gurantee that there will not be any change from Qemu side.  BTW, for the govmm side, the "string" is already there . https://github.com/kata-containers/govmm/pull/186/files#diff-60d10e04edc58668f13d9932884322892005f9b39db600185b0b08872c750579R2451

So I suggest to keep it as is.  wdyt please ? @sameo